File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Search/BedDetail.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Search

    Public Class BedConfiguration

        <JsonProperty("type")>
        Public Property Type As String

        <JsonProperty("size")>
        Public Property Size As String

        <JsonProperty("quantity")>
        Public Property Quantity As Integer
    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Search/BedDetail.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses.Search
{
    using Newtonsoft.Json;


    public class BedConfiguration
    {

        [JsonProperty("type")]
        public string Type { get; set; }

        [JsonProperty("size")]
        public string Size { get; set; }

        [JsonProperty("quantity")]
        public int Quantity { get; set; }
    }

}